package.json

```diff
@@ -18,16 +19,24 @@
     "prepack": "nuxt-module-build",
     "dev": "nuxi dev playground",
     "dev:build": "nuxi build playground",
-    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground"
+    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
+    "release": "npm run test && changelogen --release && git push --follow-tags && npm publish",
+    "lint": "eslint .",
+    "unit": "vitest run",
+    "unit:watch": "vitest watch",
```

Comment:
Our tests are not unit. Maybe simplify to `test` / `test:dev` for watch?

Reply:
That's very true, yes let's do that

Reply:
Done in [62a36ad](https://github.com/nuxt/starter/pull/216/commits/62a36adf88f82ef04756b528cfa91c11bdc4eb46)